### PR TITLE
Add my account URL to applications section

### PR DESCRIPTION
**Pull request by DonOmalVindula**
### Purpose
$subject

### Related Issues
- None

### Checklist
- [ ] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Related PRs
- None

### Security checks
- [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/applications/pages/applications.tsx [hunk 1]
```diff
@@ -276,6 +285,75 @@ const ApplicationsPage: FunctionComponent<ApplicationsPageInterface> = (
...
+     */
+    const renderTenantedMyAccountLink = (): ReactElement => {
+        if (AppConstants.getTenant() === AppConstants.getSuperTenant()) {
+            return null;
+        }
+
+        return (
+            <EmphasizedSegment
+                className="mt-0 mb-5"
+                data-testid={ `${ testId }-consumer-account-link` }
```
Comment (brionmario) [inline apps/console/src/features/applications/pages/applications.tsx:301]: Use the component id instead.
```suggestion
                data-testid={ `${ testId }-consumer-account-link` }
```

File: modules/theme/src/themes/default/assets/images/icons/solid-icons/my-account-icon.svg [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+-->
+
+<svg xmlns="http://www.w3.org/2000/svg" width="35" height="35" viewBox="0 0 35 35">
```
Comment (brionmario) [inline modules/theme/src/themes/default/assets/images/icons/solid-icons/my-account-icon.svg:19]: Please check if this is formatted with  `4` spaces

**Review by brionmario — APPROVED** [review]

File: apps/console/src/features/applications/pages/applications.tsx (hunk 1 above)
Comment (DonOmalVindula) [reply apps/console/src/features/applications/pages/applications.tsx]: Fixed with c2cc6d9fb5a4ad0937b45edc1fe14ea5f66fb6d3

File: modules/theme/src/themes/default/assets/images/icons/solid-icons/my-account-icon.svg (hunk 2 above)
Comment (DonOmalVindula) [reply modules/theme/src/themes/default/assets/images/icons/solid-icons/my-account-icon.svg:19]: Fixed with c2cc6d9fb5a4ad0937b45edc1fe14ea5f66fb6d3

**Pull request merged**